components/brave_wallet_ui/page/screens/onboarding/restore-from-recovery-phrase/restore-from-recovery-phrase.tsx

```diff
@@ -301,7 +300,6 @@ export const OnboardingRestoreFromRecoveryPhrase = ({
         onChange={onPhraseInputChanged}
         onPaste={clearClipboard}
         onKeyDown={handleKeyDown}
-        autoComplete='off'
```

Comment:
I was told by the security team that this is needed

Reply:
Needed for testing this protection, indeed. We can revert it just before the merge.